Sources/Shared/Models/PublishDestination.swift

```diff
@@ -66,17 +67,29 @@ extension PublishStatus: CustomStringConvertible {
         }
     }
     /// The URL of the post on a given platform.
-    @objc public var thirdPartyPostURL: String?
+    public var thirdPartyPostURL: String?
     
     /// The ID of the post on a given platform.
-    @objc public var thirdPartyPostID: String?
+    public var thirdPartyPostID: String?
+
+    /// The number of views this post has, as reported by the third party platform.
+    public var thirdPartyViewCount: NSNumber?
+
+    /// The number of likes (or equivalent) this post has, as reported by the third party platform.
+    public var thirdPartyLikeCount: NSNumber?
+
+    /// The number of comments this post has, as reported by the third party platform.
+    public var thirdPartyCommentCount: NSNumber?
```

Comment:
In responses still doesn't exist these values. Waiting api updates for re-check it

Reply:
@antonklevets I added these based on the updated API spec, but you're right that these haven't been implemented on the API yet. It sounds like they may be available by end of day today, but it might be worth checking back tomorrow.